ql/src/test/results/clientpositive/structin.q.out

```diff
@@ -96,10 +96,10 @@ STAGE PLANS:
       Map Operator Tree:
           TableScan
             alias: t11_n1
-            filterExpr: (struct(id,lineid)) IN (const struct('1234-1111-0074578664','3'), const struct('1234-1111-0074578695',1)) (type: boolean)
```

Comment:
Yes, some of these are just hidden by Hive implementation of the operator.
Bonus point because stricter typing on the parameters made the predicate vectorizable :)